### PR TITLE
sshfs: new package at 3.7.0

### PR DESCRIPTION
**Pull request by trbednarzyk**
Please fill out the template.

Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package

SSHFS allows you to mount a remote filesystem using SFTP and FUSE.

## Installed manifest of package

(`cat /var/db/kiss/installed/sshfs/manifest`)

```
/var/db/kiss/installed/sshfs/version
/var/db/kiss/installed/sshfs/sources
/var/db/kiss/installed/sshfs/manifest
/var/db/kiss/installed/sshfs/depends
/var/db/kiss/installed/sshfs/checksums
/var/db/kiss/installed/sshfs/build
/var/db/kiss/installed/sshfs/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/sbin/mount.sshfs
/usr/sbin/mount.fuse.sshfs
/usr/sbin/
/usr/bin/sshfs
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by dylanaraps** [comment]
The manifest shows `sbin`. This should be `/usr/bin`.

**Comment by trbednarzyk** [comment]
New manifest:
```
/var/db/kiss/installed/sshfs/version
/var/db/kiss/installed/sshfs/sources
/var/db/kiss/installed/sshfs/manifest
/var/db/kiss/installed/sshfs/depends
/var/db/kiss/installed/sshfs/checksums
/var/db/kiss/installed/sshfs/build
/var/db/kiss/installed/sshfs/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/sshfs
/usr/bin/mount.sshfs
/usr/bin/mount.fuse.sshfs
/usr/bin/
/usr/
```

**Comment by trbednarzyk** [comment]
Don't merge this yet, after changing to `/usr/bin` it doesn't appear to be working. When I try to mount an ssh connection I get the error `fusermount3: mount failed: Operation not permitted` now.

**Comment by trbednarzyk** [comment]
Don't merge this yet, after changing to `/usr/bin` it doesn't appear to be working. When I try to mount an ssh connection I get the error `fusermount3: mount failed: Operation not permitted` now.

**Comment by trbednarzyk** [comment]
As it turns out, it was a permission issue with a file in the fuse package. Everything seems to be working now.

File: community/sshfs/post-install [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+ln -sf /usr/bin/sshfs /usr/bin/mount.sshfs
+ln -sf /usr/bin/sshfs /usr/bin/mount.fuse.sshfs
```
Comment (dylanaraps) [inline community/sshfs/post-install:4]: This needs to happen in `build`.

Example: `ln -s /usr/bin/sshfs "$1/usr/bin/mount.sshfs"`

File: community/sshfs/build [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+sed -i '15,19d' utils/install_helper.sh
```
Comment (dylanaraps) [inline community/sshfs/build:3]: What does this do? A small comment explaining why might be handy here.

**Pull request merged**